src/pages/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Categories() {
+  return <div>Under Construction</div>;
+}
+
+export default Categories;
```

Comment:
- [ ] Please relocate the file to components folders. no need for a page folder.